src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -234,8 +235,8 @@ private void PendBindingInformation(ConnectionInformation connInfo)
         {
             Debug.Assert(this.qualityProfileMap != null, "Initialize was expected to be called first");
 
-            var binding = this.serviceProvider.GetService<ISolutionBindingSerializer>();
-            binding.AssertLocalServiceIsNotNull();
+            var bindingSerializer = this.serviceProvider.GetService<IConfigurationProvider>();
+            bindingSerializer.AssertLocalServiceIsNotNull();
```

Comment:
This was the final place that was still requesting the _ISolutionBindingSerializer_ service.
The _IConfigurationProvider_ now supports saving data so this method can use that instead.